man/ronin-encode.1.md

```diff
@@ -88,25 +88,25 @@ Encodes each character of the given data into a variety of formats.
 
 ## EXAMPLES
 
-Encode a string to base 16:
+Encode a string `hello_world` to base 16:
```

Comment:
You don't have to put an underscore in the string. Also might look better as `"hello world"`.